src/install/ibm-mq/linux/install-linux.mdx

```diff
@@ -0,0 +1,35 @@
+---
+componentType: default
+headingText: Install the IBM MQ monitoring integration
+---
+
+1. Change directory to the integrations configuration folder by running:
```

Comment:
These looks good! The big issue here though, is these instructions refer to files that the user would not have at this point. The user needs to actually download the integration files reffered here. A bit hard to explain, but basically the users needs to "download" the integration by running a command. Once the command is run, the files used below appear.

See this step in the MongoDB doc: https://docs.newrelic.com/install/mongodb/#download-using-apt-4